src/en/manhwahentaime/src/eu/kanade/tachiyomi/extension/en/manhwahentaime/ManhwahentaiMe.kt

```diff
@@ -32,5 +38,124 @@ class ManhwahentaiMe : Madara("Manhwahentai.me", "https://manhwahentai.me", "en"
 
         return chapterElements.map(::chapterFromElement)
     }
+
+    override fun popularMangaRequest(page: Int): Request =
+        if (useLoadMoreRequest()) {
+            loadMoreRequest(page, popular = true)
+        } else {
+            GET("$baseUrl/$mangaSubString/${searchPage(page)}?m_orderby=trending", headers)
+        }
+
+    override fun latestUpdatesRequest(page: Int): Request =
+        if (useLoadMoreRequest()) {
+            loadMoreRequest(page, popular = false)
+        } else {
+            GET("$baseUrl/home/${searchPage(page)}", headers)
+        }
+
+    override fun searchRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$baseUrl/${searchPage(page)}".toHttpUrl().newBuilder()
+        url.addQueryParameter("s", query)
+        url.addQueryParameter("post_type", "wp-manga")
+        filters.forEach { filter ->
+            when (filter) {
+                is AuthorFilter -> {
+                    if (filter.state.isNotBlank()) {
+                        url.addPathSegments("manga-author/${filter.state.replace(" ", "-")}")
+                    }
+                }
+                is ArtistFilter -> {
+                    if (filter.state.isNotBlank()) {
+                        url.addQueryParameter("artist", filter.state.replace(" ", "-"))
+                    }
+                }
+                is YearFilter -> {
+                    if (filter.state.isNotBlank()) {
+                        url.addPathSegments("webtoon-release/${filter.state}")
```

Comment:
if they are incompatible with each other then return a request for them individually (when encountering an enabled filter).
like on author.isNotBlank() -> return the request
on year.isNotBlank() -> return the request